changelog/6423.bugfix.rst

```diff
@@ -0,0 +1 @@
+Adds units (dimensionless units) to the quality columns in `XRSTimeSeries`.
```

Comment:
```suggestion
Adds units (dimensionless units) to the quality columns in `.XRSTimeSeries`.
```
It doesn't seem to link automatically but didn't raise a warning. I need to have a look at that.